src/molecules/side-menu.module.css

```
@@ -0,0 +1,92 @@
+.sideMenu {
+    position: fixed;
+    top: 0;
+    left: 0;
+    width: 22.5rem;
+    height: 100%;
+    background-color: var(--surface-color);
+    transform: translateX(-100%);
+    transition: transform 0.3s ease-in-out;
+    box-shadow: 0 4px 8px rgba(0, 0, 0, 0.1);
+    display: flex;
+    flex-direction: column;
+    border-top-right-radius: 1rem;
+    border-bottom-right-radius: 1rem;
+    padding-top: 1rem;
+  }
+  
+.sideMenu[data-open="true"] {
+    transform: translateX(0);
+}
+.sideMenu[data-open="false"] {
+  transform: translateX(-100%);
+}
+  .overlay {
+    position: fixed;
+    top: 0;
+    left: 0;
+    width: 100%;
+    height: 100%;
+    background-color: rgba(0, 0, 0, 0.2);
+    z-index: -1;
+  }
+  
+  .topBar {
+    padding: 1.5rem;
+    display: flex;
+    justify-content: flex-end;
```

```suggestion  
  align-items: end;
  justify-content: space-between;
```
タイトルを入れることも加味するとこうしたほうがよいでしょう.